packages/components/src/Workflow/_docs/Workflow.mdx

```diff
@@ -15,11 +16,17 @@ import * as WorkflowHeader from "./WorkflowHeader.stories"
 
 <KaioNotification />
 
+<InlineNotification type="negative" persistent>
+  {`This version of the Workflow is deprecated and will be updated in the next major release.
+  Import from "@kaizen/components/future" for the latest version.`}
```

Comment:
```suggestion
  {`This version of the Workflow is deprecated and will be removed in the next major release.
  Import from "@kaizen/components/future" for the latest version.`}
```

Reply:
heheh um'd and ah'd on that exact word 😆